app/lib/redis_configuration.rb

```diff
@@ -45,6 +45,9 @@ def url
   private
 
   def raw_connection
-    Redis.new(url: url, driver: :hiredis)
+    Redis.new(
+      url: url,
+      ssl_params: { verify_mode: OpenSSL::SSL::VERIFY_NONE },
```

Comment:
Is this required? I tested removing hiredis and just leaving `Redis.new(url: url)` and, using `rediss://` works pretty well.

Reply:
If the Redis server has a verifiable SSL certificate that can be chained back to a trusted CA certificate, the `verify_mode` isn't necessary. Heroku Data for Redis, which I use, currently only presents a self-signed certificate which isn't verifiable.